pyaedt/icepak.py

```diff
@@ -370,6 +411,30 @@ def create_network_blocks(self, input_list, gravity_dir, top, assign_material=Tr
                     networks.append(out)
         return networks
 
+    @aedt_exception_handler
+    def assign_surface_monitor(self, face_name, monitor_type="Temperature", monitor_name=None):
+        """Assign a Surface monitor
+
+        Parameters
+        ----------
+        face_name : str
+        monitor_type : str, optional
+        monitor_name : str, optional
+
+        Returns
+        -------
+        bool
+        """
+        if not monitor_name:
+            monitor_name =generate_unique_name("Monitor")
+        oModule = self.odesign.GetModule("Monitor")
+        oModule.AssignFaceMonitor(
+            [
+                "NAME:"+ monitor_name,
+                "Quantities:="	, [monitor_type],
+                "Objects:="		, [face_name]
```

Comment:
It's bad practice to include whitespace like this.

Reply:
I agree, that comes from native API. I'll get rid of that

Reply:
Changed